WordPress/Classes/Categories/UIImageView+Gravatar.m

```diff
@@ -81,6 +83,13 @@ - (NSURL *)gravatarURLForEmail:(NSString *)email withSize:(NSInteger)size gravat
     return [NSURL URLWithString:gravatarUrl];
 }
 
+- (NSURL *)siteIconURLForSiteIconUrl:(NSString *)path
+{
+    NSInteger size = [self sizeForBlavatarDownload];
+    NSString *siteIconUrl = [NSString stringWithFormat:@"%@?w=%d&h=%d", path, size, size];
```

Comment:
The blavatar code already had this problem, but what happens if the photon URL already has sizing parameters?